app/models/manageiq/providers/openshift/container_manager_mixin.rb

```diff
@@ -36,7 +36,7 @@ def openshift_connect(hostname, port, options)
       Kubeclient::Client.new(
         raw_api_endpoint(hostname, port, '/oapi'),
         options[:version] || api_version,
-        :ssl_options    => { :verify_ssl => verify_ssl_mode },
+        :ssl_options    => options[:ssl_options],
```

Comment:
TODO: if not given this passes nil, which crashes kubeclient (it assumes `@ssl_options` is always a hash, the default hash is only used if :ssl_options it not passed at all to constructor).
This only affects calls `get_hostname_from_routes`, and will be masked by the UI PR which starts passing :ssl_options from `get_hostname_from_routes`, but I should fix this.

[same in `kubernetes_connect`]